ContentPatcher/docs/i18n/zh/token-strings-api.md

```diff
@@ -0,0 +1,192 @@
+﻿← [README](README.md)
+
+此文档帮助SMAPI模组作者在自己的模组中使用Content Patcher的令牌字符串。
+
+**若需要添加给其他内容包使用的自定义令牌，请参阅[扩展性API](extensibility.md)。其他信息请参见[主README](README.md)**
+
+**🌐 其他语言： [en (English)](../../token-strings-api.md)。**
+
+## 目录
+* [概述](#overview)
+* [访问API](#access-the-api)
+* [解析令牌字符串](#parse-token-strings)
+* [管理令牌字符串](#manage-token-strings)
+* [注意事项](#caveats)
+* [参见](#see-also)
+
+## 概述<a name="overview"></a>
+Content Patcher有一个令牌系统(author-guide/tokens.md)，允许内容包组成复杂并遵守上下文的字符串。
+```js
+""
+"My favorite season is {{Season}}." // If a save is loaded, {{Season}} will be replaced with the current season.
+```
+
+其他SMAPI模组也可以使用此系统，通过API解析一个包含令牌的字符串来获得一个'托管令牌字符串'对象，之后使用此对象来管理令牌字符串。
+
+## 访问API<a name="access-the-api"></a>
+
+访问API的步骤为：
+
+1. 将Content Patcher设为[`manifest.json`中的**必要**依赖](https://stardewvalleywiki.com/Modding:Modder_Guide/APIs/Manifest#Dependencies):
+   ```js
+   "Dependencies": [
+      { "UniqueID": "Pathoschild.ContentPatcher", "MinimumVersion": "2.7.0" }
+   ]
+   ```
+2. 在你模组的`.csproj`里添加对Content Patcher DLL的引用。将此引用设置为`Private="False"`，确保它不被复制到你的模组文件夹中：
+_译者注：这段似乎没更新，使用API只需将IContentPatcherAPI.cs和IManagedTokenString.cs复制到你自己模组中即可，_
+   ```xml
+   <ItemGroup>
+     <Reference Include="ContentPatcher" HintPath="$(GameModsPath)\ContentPatcher\ContentPatcher.dll" Private="False" />
+   </ItemGroup>
+   ```
+3. 在你的模组代码中（如[`GameLaunched`事件](https://zh.stardewvalleywiki.com/模组:制作指南/APIs/Events#GameLoop.GameLaunched)中，获取Content Patcher的API：
+   get a reference to Content Patcher's API:
+   ```c#
+   var api = this.Helper.ModRegistry.GetApi<ContentPatcher.IContentPatcherAPI>("Pathoschild.ContentPatcher");
+   ```
+
+## 解析令牌字符串<a name="parse-token-strings"></a>
+**注意:** 使用API前请阅读[_注意事项_](#caveats)。
+
+有API以后你就可以解析令牌字符串了。
+
+1. 创建一个需解析的字符串。这个字符串可包含[令牌](author-guide/tokens.md)例如：
+   ```c#
+   string rawTokenString = "The current time is {{Time}} on {{Season}} {{Day}}, year {{Year}}.";
+   ```
+2. 用API将条件解析为一个`IManagedTokenString`对象。其中的`formatVersion`对应[作者指南中描述的`Format`字段](author-guide.md#overview)，用于维持跟未来的Content Patcher版本兼容。
+
+   ```c#
+   var tokenString = api.ParseTokenString(
+      manifest: this.ModManifest,
+      rawValue: rawTokenString,
+      formatVersion: new SemanticVersion("2.7.0")
+   );
+   ```
+3. 从`Value`属性中获取结果，例如：
+   ```cs
+   tokenString.UpdateContext();
+   string value = tokenString.Value; // The current time is 1430 on Spring 5, year 2.
+   ```
+
+如果你想使用其他SMAPI模组添加的令牌，你可以用`assumeModIds`来指定所安装的模组id。你不需要把自己ID，必要依赖的ID,和任何在条件字段中以`HasMod`指定的Id添加到`assumeModIds`中。
```

Comment:
The `HasMod` bit seems to be new. I suggest making sure any non-translation improvements are in the original docs too, so the info is consistent.